.github/workflows/docker.yml

```diff
@@ -15,6 +17,14 @@ jobs:
         password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
     - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v1
+    - name: Generate container metadata
+      id: meta
+      uses: docker/metadata=action@v4
+      with:
```

Comment:
I think we should follow the basic example of pushing the tags and not explicitly follow semver (the GitHub tags are already semver). That means we should get rid of this `tags` definition. The tag is automatically used from the GitHub tag.

Additionally, it looks like we need to include `images: tdewolff/minify` here.

Reply:
I have no idea; could be; as for the tags, I think what this action also tries to do, is do  a 'on push of `v3.1.2`, also push `v3.1` and `v3`, which is why you'd define them.